autometrics/build.rs

```diff
@@ -0,0 +1,30 @@
+use cfg_aliases::cfg_aliases;
+
+pub fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
happens automatically if set in cargo.toml

Reply:
This just means that it won't be run _unless_ build.rs changes (i.e. it won't be re-run for changes to any other files)